src/window/action_popup.h

```diff
@@ -0,0 +1,12 @@
+#ifndef WINDOW_ACTION_DIALOG_H
+#define WINDOW_ACTION_DIALOG_H
+
+#include <stdint.h>
```

Comment:
I don't think you need `stdint.h` here. If you need it on `action_popup.c`, add it there instead, please.